lib/graphql/language/nodes.rb

```diff
@@ -66,7 +66,7 @@ def initialize_copy(other)
 
         # @return [Symbol] the method to call on {Language::Visitor} for this node
         def visit_method
-          raise NotImplementedError, "#{self.class.name}#visit_method shold return a symbol"
+          raise GraphQL::RequiredImplementationMissingError, "#{self.class.name}#visit_method shold return a symbol"
```

Comment:
```suggestion
          raise GraphQL::RequiredImplementationMissingError, "#{self.class.name}#visit_method should return a symbol"
```

Unrelated to your change, but I couldn't help noticing it! 🙈

Reply:
Nevermind, I thought I could commit changes from the comment, but it's not working 🤷‍♂

Reply:
Sorry, I didn't notice this typo 🤦‍♂ 
I can prepare a fix for that as well 😄